### ISSUE TITLE
pip install throwing dependency error

### ISSUE THREAD

**Issue by sefabey** [narkopolo/fb_friend_list_scraper#1]
Am getting a dependency error. 

```
$ pip install fb_friend_list_scraper
Collecting fb_friend_list_scraper
  Using cached fb_friend_list_scraper-0.3.3-py3-none-any.whl (19 kB)
Collecting pyautogecko==0.1.3
  Downloading pyautogecko-0.1.3-py3-none-any.whl (5.5 kB)
Collecting tqdm==4.43.0
  Downloading tqdm-4.43.0-py2.py3-none-any.whl (59 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 59.1/59.1 kB 1.3 MB/s eta 0:00:00
Collecting fb_friend_list_scraper
  Using cached fb_friend_list_scraper-0.3.2-py3-none-any.whl (19 kB)
ERROR: Cannot install fb-friend-list-scraper==0.3.2 and fb-friend-list-scraper==0.3.3 because these package versions have conflicting dependencies.

The conflict is caused by:
    fb-friend-list-scraper 0.3.3 depends on rich==12.2.0selenium-wire==4.6.3
    fb-friend-list-scraper 0.3.2 depends on rich==12.2.0selenium-wire==4.6.3

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts```

**Comment by narkopolo** [comment]
I'll look into this ASAP

**Comment by n0kovo** [comment]
Fixed in e41c1c5192e80fdfa9b2bb2aa83bd627a64d7c0f.

Turns out I was missing a comma in the dependency dictionary in `setup.py`, so PIP was looking for a package called `rich==12.2.0selenium-wire==4.6.3` 🥴

I've yanked the faulty version from PyPI so it should install without problems now.
Thanks for letting me know!

**Comment by sefabey** [comment]
Thanks for sorting!
